### DISCUSSION TITLE
Avorion Server not starting

### DISCUSSION THREAD

**Discussion by AlmHurricane** [GameServerManagers/LinuxGSM#3314] [Q&A] [answered]
Distro: Debian 8

Everytime I try to start my Avorion server with "./avserver st" it will boot up and immediately shut down again. Logs only show the following: 
![dump](https://user-images.githubusercontent.com/15094871/109434876-a110c780-7a17-11eb-87b8-610a472dbf78.png)

What did I mess up? xD

**Comment by EugeneGGHQ** [comment, marked as answer]
`ip: --datapath`

That value looks suspicious to me - double check your server config and make sure the ip references an address with a valid format.

**Reply by rconjoe** [reply]
--datapath is a steamcmd start parameter for Avorion indicating where world files are stored. It's found directly after --ip in the default config:
`startparameters="--galaxy-name ${selfname} --ip ${ip} --datapath ${avdatapath} ${admincmd}"`
**Do you have an ip set in your lgsm config for that instance?** If not, I wonder if it's grabbing "--datapath" since it's the next string.

**Comment by NeoxAce** [comment]
I have the problem when I start the avserver with ./avserver start it always creates an .avorion folder under /home/avserver/.avorion and when I edit the server.ini ect.. it resets everything and creates everything again but bein ausfüren says the server name I created but it doesn't appear in the server list and under Join by IP it says A Avorion server

**Comment by NeoxAce** [comment]
and I have already reinstalled the vServer 6 times and done everything as it is in linuxgsm
